app/src/main/java/com/duckduckgo/app/browser/WebDataManager.kt

```diff
@@ -95,10 +102,15 @@ class WebViewDataManager @Inject constructor(
     companion object {
         private const val WEBVIEW_DATA_DIRECTORY_NAME = "app_webview"
         private const val WEBVIEW_DEFAULT_DIRECTORY_NAME = "app_webview/Default"
+        private const val DATABASES_DIRECTORY_NAME = "databases"
 
         private val WEBVIEW_FILES_EXCLUDED_FROM_DELETION = listOf(
             "Default",
             "Cookies"
         )
+
+        private val HTTP_AUTH_FILES_TO_DELETE = listOf(
+            "http_auth.db-journal"
+        )
```

Comment:
This assumes the `http_auth` db will always use a rollback journal, which might not be the case always.

We could also try to set the journal mode of the db to `DELETE`. That should delete the journal every time a transaction happens.
AFAIR from when I was doing PR #1066, this DB journal mode is set to `TRUNCATE`, and that might be why the journal file is not deleted after each transaction

I don't think we should worry about performance here, we're not doing intensive read/writes in the db

Reply:
Agreed and done!